src/events.rs

```diff
@@ -194,6 +201,10 @@ impl<T: System> EventsDecoder<T> {
                         // PhantomData is size 0
                         return Ok(())
                     }
+                    if name.contains("DispatchError") {
+                        let error = DispatchError::decode(input)?;
+                        return Err(error.into())
+                    }
```

Comment:
Not sure if this makes any sense, not had a `DispatchError::Other` since you updated to alpha.7

Reply:
Probably because I increased the gas limit of instantiate which no longer causes the `DispatchError` to be thrown. Still we need to handle contract call errors - could you remove this from this PR we can fix it separately via #103